source/WebConfig/webconfig_internal.c

```diff
@@ -1404,6 +1419,7 @@ void* processWebConfigNotification()
 		msg = notifyMsg;
 		if(msg !=NULL)
 		{
+			pthread_mutex_unlock (&notify_mut);
```

Comment:
msg and notifyMsg are modified below, so unlock cannot be done here.

Reply:
Incorporated.